tests/regression/package/regression_package_base.py

```diff
@@ -0,0 +1,77 @@
+import os
+import uuid
+import json
+import tempfile
+import time
+from pathlib import Path
+from unittest import TestCase
+
+import boto3
+
+
+class PackageRegressionBase(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.region_name = os.environ.get("AWS_DEFAULT_REGION")
+        cls.bucket_name = str(uuid.uuid4())
+        cls.test_data_path = Path(__file__).resolve().parents[2].joinpath("integration", "testdata", "package")
+
+        # Create S3 bucket
+        s3 = boto3.resource("s3")
+        cls.s3_bucket = s3.Bucket(cls.bucket_name)
+        cls.s3_bucket.create()
+
+        # Given 3 seconds for all the bucket creation to complete
+        time.sleep(3)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.s3_bucket.objects.all().delete()
+        cls.s3_bucket.delete()
+
+    def base_command(self, base):
+        command = [base]
+        if os.getenv("SAM_CLI_DEV") and base == "sam":
+            command = ["samdev"]
+        elif base == "aws":
```

Comment:
Does `aws` work on windows?

Reply:
we are installing via pip, so the executable is aws.exe and should work.

Reply:
Currently we check for both .exe and .cmd for windows. https://github.com/awslabs/aws-sam-cli/blob/develop/samcli/lib/samlib/cloudformation_command.py#L43